js/classes.js

```diff
@@ -260,6 +257,15 @@ class Item {
         return getEffectDescription(this.baseData.effects, 1);
     }
 
+    /**
+     * @param {EffectType} effectException
+     * @return {string}
+     */
+    getEffectDescriptionExcept(effectTypeException) {

```

Comment:
Wofür brauchst du das denn? 🤔

Reply:
Danger ist ein effect. Aber Danger steht im UI in einer eigenen Spalte.

Reply:
Ahjo, okay, sorry 😅 ergibt Sinn von der Implementation